.circleci/config.yml

```diff
@@ -97,13 +97,6 @@ jobs:
           workspace: examples/lib-simple
           run-yarn: true
 
-  example-lib-complex:
-    <<: *defaults
-    steps:
-      - test:
-          workspace: examples/lib-complex
```

Comment:
I noticed all of lib-complex, including the CI is setup, is gone. Is this related to the web3 upgrade?

Reply:
Yep, lib-complex was using outdated versions of everything - it even pulled `zos-lib` and `web3@beta.37`. I spent a few hours trying to upgrade it, and eventually decided to cut it.